README.md

```diff
@@ -48,7 +48,9 @@ http {
             proxy_pass http://127.0.0.1:8080/backend;
 
             body_filter_by_lua_block {
-                require("tracer"):finish()
+                if ngx.eof then
```

Comment:
@spacewander Could you apply this change to https://github.com/apache/skywalking-nginx-lua/blob/master/examples/nginx.conf#L78 too?

That file is for local closed-loop test, it makes the dev team doesn't require the SkyWalking backend.

Reply:
@spacewander is `ngx.eof` the same as `ngx.arg[2]`?  Is it a hidden variable?

Reply:
@wu-sheng @moonming 
Sorry, it's my mistake. It should be `ngx.arg[2]` indeed.

Reply:
Fixed in the new commit.

Reply:
What is `ngx.arg[2]`? Could you explain a little more? If `the eof means the end of the response body data`, why isn't it suitable?

Reply:
@wu-sheng 
`ngx.eof` is a function which marks the end of the body, but not the mark.
In `body_filter_by_lua*` phase, the `ngx.arg[1]` returns the body data and `ngx.arg[2]` returns the mark itself (a metamethod is called under the hover).  
So in my previous commit, use `ngx.eof` (a function) is incorrect, `ngx.arg[2]`(the mark) should be used instead.